releasenotes/notes/add-cmap-componets-7ed56cdf294150f1.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Added support to the :class:`~.CouplingMap` object to have a disjoint
+    connectivity. Previously, a :class:`~.CouplingMap` could only be
+    constructed if the graph was connected. This will enable using
+    :class:`~.CouplingMap` to represent hardware with disjoint qubits.
+  - |
+    Added a new method :meth:`.CouplingMap.components` which
```

Comment:
I didn't think "components" was particularly graph-specific terminology, but I'm also fine with either segments or partitions.

Reply:
I don't really have a strong opinion on the naming, it makes sense to me. I originally called this `get_component_subgraphs` and renamed it per @jakelishman's suggestion. So I'll defer to whatever you think is best here.

Reply:
Per the consensus on our offline poll I renamed the method `connected_components()` in: https://github.com/Qiskit/qiskit-terra/pull/9710/commits/0936fb044b89de21cec8728c20519f5a6986c241

and fixed the release note here in: https://github.com/Qiskit/qiskit-terra/pull/9710/commits/89579973e3a36a423bc81e3537a1db6f6cc7f32f (because I missed it in the initial commit)